pino.js

```diff
@@ -44,6 +57,10 @@ function pino (opts, stream) {
     buf: ''
   }
 
+  if (opts.enable === false) {
```

Comment:
ok sure but.. it should be `enabled` rather than `enable` since enabled is a state and enable is action and options configure state rather than trigger actions (.... I'm sooooo picky today)